### ISSUE TITLE
ga-wdi-boston/ember-resources-diagnostic

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/ember-resources-diagnostic#1]

**Comment by jrhorn424** [comment]
This took the whole morning, if memory serves. Developers got an hour to work on the challenge, and walking through the solution took about two hours.

Not sure if this should be a challenge or a diagnostic in the future. If it's a diagnostic, we should consider making it shorter, and perhaps making it a markdown-based set of questions. Having an external dependency, like an API, increases the surface area for maintenance between each cohort and may not be appropriate for a challenge or diagnostic. We should probably discuss further.

**Comment by MattB-GA-Boston** [comment]
@jrhorn424 
Yeah, I wanted to try to evaluate their actual skill with Ember, but it just ended up being a mess. Maybe a set of questions would be better.

Originally, I wasn't planning on having students hit an actual API, just build the necessary files in Ember. 
But when we started, they complained that they couldn't see what they were doing - that's why I tried to create an API at the last minute.
